README.md

```diff
@@ -59,6 +59,7 @@ this is a webpack loader that can translate your code and generate bundle per ea
 ### query params:
 - `funcName`: change default function name (default is `__()`)
 - `quotes`: choose how to generates translated strings in code (default is `''`)
+- `lng`: choose the language of which bundle (default is undefined, if we pass nothing will use on the init)
```

Comment:
I don't quite understand this sentence. Please change it to something like `set i18next language (useful when having webpack config per language)`

Reply:
Done my friend!